MapboxSearch/sample/src/main/java/com/mapbox/search/sample/api/OfflineSearchJavaExampleActivity.java

```diff
@@ -81,7 +81,7 @@ public void onIndexChange(@NonNull OfflineIndexChangeEvent event) {
                     Log.i("SearchApiExample", tileRegionId + " was successfully added or updated");
 
                     searchRequestTask = searchEngine.search(
-                        "Cafe",
+                        "2011 15th Street Northwest, Washington, District of Columbia",
```

Comment:
What cause this changes? We can't find expected point by `Cafe` query?

Reply:
Actually, it can find something. But at the moment our SDK doesn't provide offline POI search so I wanted to make it more clear in our samples.